lib/driver.js

```diff
@@ -408,12 +409,11 @@ class XCUITestDriver extends BaseDriver {
   async executeCommand (cmd, ...args) {
     log.debug(`Executing command '${cmd}'`);
     if (cmd === 'receiveAsyncResponse') {
-      return await this.receiveAsyncResponse(...args);
+      return await this._executeCommandWithTimeout(B.promisify(this.receiveAsyncResponse), 'receiveAsyncResponse', ...args);
```

Comment:
yep, but one cannot call **timeout** method on that promise. That is why I need to transform it into bluebird promise first. Do you see another approach?